biggraphite/drivers/elasticsearch.py

```diff
@@ -566,10 +576,31 @@ def has_metric(self, metric_name):
     def get_metric(self, metric_name, touch=False):
         """See the real Accessor for a description."""
         super(_ElasticSearchAccessor, self).get_metric(metric_name, touch=touch)
+        
         metric_name = ".".join(_components_from_name(metric_name))
-        # FIXME This hack make it possible to use glob_metric_names with bgutil list
-        # This need to be really implemented.
-        return self.make_metric(metric_name, bg_accessor.MetricMetadata())
+
+        metric = self.__get_metric(metric_name)
+
+        if touch:
+            self.__touch_metadata_on_need(metric, metric.updated_on)
+
+        return self.make_metric(
```

Comment:
could be worth reusing the timestamps here: see https://github.com/criteo/biggraphite/blob/master/biggraphite/accessor.py#L647

Reply:
Done